gophish_cloud_init.tf

```diff
@@ -132,6 +132,30 @@ data "cloudinit_config" "gophish_cloud_init_tasks" {
     merge_type   = "list(append)+dict(recurse_array)+str()"
   }
 
+  # Install certificate for Gophish.
+  part {
+    content = templatefile(
+      "${path.module}/cloud-init/install-certificates.py", {
```

Comment:
So small nitpick, but with this change in usage should this file be renamed to `install-certificates.tpl.py`?

Reply:
Please see commit 7a5f278.